webextension/background/main.js

```diff
@@ -62,5 +63,19 @@ window.main = (function () {
     chrome.tabs.create({url: backend + "/shots"});
   });
 
+  communication.register("openShot", ({url, copied}) => {
```

Comment:
Actually, after seeing the PR for the `/creating` page I should change this to not open the tab.